src/Radicle/Internal/Effects/Capabilities.hs

```diff
@@ -137,8 +140,46 @@ instance ReadFile IO where
 instance {-# OVERLAPPABLE #-} ReadFile m => ReadFile (Lang m) where
     readFileS = lift . readFileS
 
+class (Monad m) => FileModule m where
+    fileModuleS :: Text -> m (Either Text Text)
+instance FileModule IO where
+    fileModuleS file = do
+        x <- findModule $ toS file
+        case x of
+            Nothing -> pure . Left $ "Could not find file: " <> file
+            Just f  -> pure . Right $ toS f
+instance FileModule (InputT IO) where
+    fileModuleS = lift . fileModuleS
+instance {-# OVERLAPPABLE #-} FileModule m => FileModule (Lang m) where
+    fileModuleS = lift . fileModuleS
+
 putStrLnS :: (Stdout m) => Text -> m ()
 putStrLnS t = putStrS t >> putStrS "\n"
 
 modifyEnvS :: (GetEnv m r, SetEnv m r) => (Env r -> Env r) -> m ()
 modifyEnvS f = getEnvS >>= setEnvS . f
+
+-- | Importing works as follows:
+--
+--    (1) If RADPATH exists, search there
```

Comment:
Could you please add that `RADPATH` has the same format as PATH with multiple paths separated by a colon.

Reply:
Done